Tasks/ANT/package-lock.json

```diff
@@ -295,7 +295,7 @@
     "minimatch": {
       "version": "3.0.4",
       "resolved": "https://registry.npmjs.org/minimatch/-/minimatch-3.0.4.tgz",
-      "integrity": "sha512-yJHVQEhyqPLUTgt9B83PXu6W3rx4MvvHvSUvToogpwoGDOUQ+yDrR0HRot+yOCdCO7u4hX3pWft6kWBBcqh0UA==",
+      "integrity": "sha1-UWbihkV/AzBgZL5Ul+jbsMPTIIM=",
```

Comment:
@ericsciple @stephenmichaelf : What is the guidance on committing the package-lock.json files when we are actually not changing the dependencies?

Reply:
I spoke with Eric regarding this and he said that it is fine. The change in integrity SHA may be due to different npm versions.